lib/pr_checker.js

```diff
@@ -523,6 +523,32 @@ export default class PRChecker {
     return true;
   }
 
+  async checkCommitsAfterReviewOrLabel() {
+    if (this.checkCommitsAfterReview()) return true;
+
+    await Promise.all([this.data.getLabels(), this.data.getCollaborators()]);
```

Comment:
Sorry, delayed second thought: In general, why did you decide to fetch the labeled events lazily and not with the rest of the timeline events on the PR? If there is no particular reason, it looks to me like just extending the PR query would be less fragile _and_ easier to test.

Reply:
How would that change the testability? Making extra requests will make it more likely that the bot will reach its quota of requests and be rate limited, making the requests lazilly seems like a low-hanging fruit.

Reply:
With "easier to test" I didn't mean it is "hard to test" this way either, just that the work of extending the existing stubs with some values and having all the data present synchronously vs testing lazy fetching would be slightly less. Anyway, the rate limit is a fine reason. 👍